packages/editor-ui/src/plugins/i18n/index.ts

```diff
@@ -110,6 +110,25 @@ export class I18nClass {
 				});
 			},
 
+			/**
```

Comment:
@ivov can you please review this file? not sure why some  of this is needed

Reply:
we don't use that?

Reply:
sorry just asking for Ivan's input because I am not familiar with this implementation

Reply:
Hint translation working in node and credential: 

```json
{
	"nodeView": {
		"propertyName1": {
			"hint": "🇩🇪 Enter the name of the property (JSON key). Supports dot-notation."
		}
	}
}
```

<img width="317" alt="image" src="https://user-images.githubusercontent.com/44588767/150355146-ff123aeb-10e5-41fd-ad20-962daf2597fe.png">

```json
{
	"spaceId": {
		"hint": "🇩🇪 Temporary credential test <b> bold me me </b>"
	}
}
```

<img width="331" alt="image" src="https://user-images.githubusercontent.com/44588767/150356143-9b680cbf-4943-42d0-b05f-f6a481693477.png">

Suggestions:
1. `inputLabelHint` might be an inaccurate name. It is not a hint about the input label - it is a hint about the param itself, so it is equivalent to `placeholder`, hence maybe just `hint`. Not very familiar with the feature so please use your judgment.
2. The condition `if (['clientId', 'clientSecret'].includes(parameterName))` in `inputLabelDisplayName` for `credText` allows for Client ID and Client Secret to be translated once and reused across all OAuth creds params. It does not seem that hints will benefit from this - and even if they will, certainly not from Client ID and Client Secret.
3. In the comment `Display hint for a top-level param.`, I would remove `Display` as it is implied. `Display` exists in the docstring for `displayName` because it is part of the property name.
4. In the comment `Hint (hint text) for an input, whether top-level or nested.`, I would remove `(hint text)` as it does not add anything. `(tooltip text)` exists in `inputLabelDescription` because it clarifies where the description is shown, so there it is helpful.

Reply:
thanks for your suggestions, renamed `inputLabelHint ` to ` parameterHint`, updated comments, removed condition

Reply:
All the i18n methods in `credText` and `nodeText` are for a parameter, so ideally just `hint`.